### PR TITLE
Updates for configKubernetes.md

### PR DESCRIPTION
**Pull request by nalind**
This is mainly about updating the sample configuration JSON and output data to better align with v1beta3, but we also fix a few typos while we're here.

### CODE REVIEW

File: configKubernetes.md [hunk 1]
```diff
@@ -136,39 +136,37 @@ NAME           LABELS        STATUS
...
-            "id": "apache",
-            "restartPolicy": {
-                "always": {}
-            },
-            "version": "v1beta1",
-            "volumes": null
-        }
-    },
-    "id": "apache",
+    "apiVersion": "v1beta3",
```
Comment (mtrmac) [inline configKubernetes.md]: apiVersion needs to be "v1" nowadays.

File: configKubernetes.md [hunk 2]
```diff
@@ -234,19 +248,28 @@ Now that the pod is known to be running we need a way to find it.  Pods in kuber
 
 ```json
 {
-    "apiVersion": "v1beta1",
-    "containerPort": 80,
-    "id": "frontend",
+    "apiVersion": "v1beta3",
```
Comment (mtrmac) [inline configKubernetes.md]: apiVersion needs to be "v1" nowadays.

File: configKubernetes.md [hunk 3]
```diff
@@ -234,19 +248,28 @@ Now that the pod is known to be running we need a way to find it.  Pods in kuber
...
+        "selector": {
+            "name": "apache"
+        },
+        "ports": [
+            {
+                "protocol": "TCP",
+                "port": 80
+            }
+        ],
+        "publicIPs": [
```
Comment (mtrmac) [inline configKubernetes.md]: With v1, publicIPs silently fails. Using "deprecatedPublicIPs" instead works; though perhaps long-term the example should be modified to not rely on this functionality at all.

File: configKubernetes.md [hunk 4]
```diff
@@ -301,44 +324,45 @@ This should have the exact same definition of the pod as above, only now it is b
...
-                            ]
-                        }
-                    ],
-                    "id": "apache",
-                    "restartPolicy": {
-                        "always": {}
-                    },
-                    "version": "v1beta1",
-                    "volumes": null
+    "apiVersion": "v1beta3",
```
Comment (mtrmac) [inline configKubernetes.md]: apiVersion needs to be "v1" nowadays.

File: configKubernetes.md (hunk 3 above)
Comment (eparis) [inline configKubernetes.md]: deprecatedPublicIPs works?  I thought that was stopped.  In any case, I agree, should be made into a nodePort type v1 service.

**Pull request closed without merging**